amundsen_application/static/js/components/common/PopularTables/index.tsx

```diff
@@ -3,15 +3,20 @@ import * as React from 'react';
 // TODO: Use css-modules instead of 'import'
 import './styles.scss';
 
-import { POPULAR_TABLES_LABEL, POPULAR_TABLES_INFO_TEXT, POPULAR_TABLES_SOURCE_NAME } from './constants';
+import {
+  POPULAR_TABLES_LABEL,
+  POPULAR_TABLES_INFO_TEXT,
+  POPULAR_TABLES_SOURCE_NAME,
+  POPULAR_TABLES_PER_PAGE
+} from './constants';
 import InfoButton from 'components/common/InfoButton';
-import SearchList from 'components/SearchPage/SearchList';
 
 import { getPopularTables } from 'ducks/popularTables/reducer';
 import { GetPopularTablesRequest, TableResource } from 'ducks/popularTables/types';
 import { GlobalState } from 'ducks/rootReducer';
 import { bindActionCreators } from 'redux';
 import { connect } from 'react-redux';
+import ResourceList from "components/common/ResourceList";
```

Comment:
nit: single quotes

Reply:
I think my IDE auto imports with double quotes. I'll have to look at settings.

Reply:
If that doesn't work out, we can also check to see if our linter can auto fix.